.azure-pipelines/pull_request.yml

```diff
@@ -51,7 +51,7 @@ stages:
           - bash: |
               set -ex
               conda --version
-              conda env create -f scipp-developer.yml
+              conda env create -f scipp-developer-no-mantid.yml
```

Comment:
Shouldn't we just drop rename this env file to be `scipp-developer.yml` now? we have no use for Mantid in scipp directly any longer.

Reply:
👍